Source/WebInspectorUI/UserInterface/Views/FontVariationDetailsSectionRow.js

```diff
@@ -59,6 +59,7 @@ WI.FontVariationDetailsSectionRow = class FontVariationDetailsSectionRow extends
         this._variationMaxValueElement.textContent = this._formatAxisValueAsString(maximumValue);
 
         this._inputRangeElement.addEventListener("input", (event) => {
+            this.value = event.target.value;
```

Comment:
This keeps the perception to a live Fonts panel UI while a user is dragging on a slider. State is update locally and then synchronized later once the debounced update occurs.